### PR TITLE
[test] Add `reftest` helper method for browser tests. NFC

### PR DESCRIPTION
**Pull request by sbc100**
This makes it easy distinguish reference tests from other btests.

### CODE REVIEW

File: test/test_browser.py [hunk 1]
```diff
@@ -3424,7 +3421,7 @@ def test_sdl2_misc_via_object(self):
   @requires_sound_hardware
   def test_sdl2_mixer_wav(self, flags):
     shutil.copyfile(test_file('sounds/the_entertainer.wav'), 'sound.wav')
-    self.btest_exit('test_sdl2_mixer_wav.c', args=['--preload-file', 'sound.wav'] + flags)
+    self.btest_exit('test_sdl2_mixer_wav.c', args=['--profiling-funcs', '--preload-file', 'sound.wav'] + flags)
```
Comment (kripken) [inline test/test_browser.py:3424]: This changes looks unrelated?

Comment (sbc100) [reply test/test_browser.py]: reverted

**Review by kripken — APPROVED** [review]
